modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -22,6 +33,30 @@ export interface FeatureFlags {
 
 const test = mergeTests(backendPageTest);
 
+const getCompanyId = (value: boolean | ScopedFeatureFlagValue) => {
+	let companyId: number | undefined = undefined;
```

Comment:
Not sure why I cant call `Liferay.ThemeDisplay.getCompanyId()` here. It says that Liferay is not defined.